nora.gemspec

```diff
@@ -23,10 +24,10 @@ Gem::Specification.new do |spec|
 
   spec.add_development_dependency "bundler", "~> 1.10"
   spec.add_development_dependency "overcommit", "~> 0.38"
-  spec.add_development_dependency "rubocop", "~> 0.47"
+  spec.add_development_dependency "rubocop", "~> 0.49.0"
```

Comment:
The previous version of Rubocop had a vulnerability. See: https://github.com/panorama-ed/nora/network/dependencies